frontend/src/app/testResults/uploads/UploadForm.tsx

```diff
@@ -403,10 +391,13 @@ const Uploads = () => {
                 <p className="usa-process-list__heading">
                   Download the{" "}
                   <a
```

Comment:
same question here... should we just display text if there is no `spreadsheetTemplateLocation`?